changes/fix_nameConsistency.md

```diff
@@ -0,0 +1 @@
+Project name should all be "Dimsum" instead of "Dim Sum"
```

Comment:
This reads more like an outstanding issue than a description of a change, and it's ambiguous. Are they all 'Dimsum' or aren't they? I would recommend something like 'Changed references to previous name, 'Dim Sum', to 'Dimsum''.